src/coreclr/vm/codeman.cpp

```diff
@@ -1514,8 +1514,8 @@ void EEJitManager::SetCpuInfo()
         uint32_t maxVectorTLength = (maxVectorTBitWidth / 8);
         uint64_t sveLengthFromOS = GetSveLengthFromOS();
```

Comment:
This method to be adjusted to return `0` on Windows with this change as otherwise we could be in an unsupported state since it's currently hardcoded as:
```asm
;; uint64_t GetSveLengthFromOS(void);
    LEAF_ENTRY GetSveLengthFromOS
        ;; TODO-SVE: Remove the hardcoded value 128 and uncomment once CI machines are updated to use MASM 14.4 or later
        ;; rdvl    x0, 1
        mov     x0, #128
        ret     lr
    LEAF_END
```

Reply:
hhm, so this is the case where if windows is running on 256-bit machine, we shouldn't be returning 128-bits because that will falsely turn on SVE. Let me uncomment the right way to do it to see if helix machines got the assembler upgraded to understand `rdvl`.

Reply:
alright, so that seemed to be working now.